service/matching/task_queue_partition_manager_test.go

```diff
@@ -76,8 +77,10 @@ func (s *PartitionManagerTestSuite) SetupTest() {
 		useNewMatcher(config)
 	}
 
-	matchingClientMock := matchingservicemock.NewMockMatchingServiceClient(s.controller)
-	engine := createTestMatchingEngine(logger, s.controller, config, matchingClientMock, registry)
+	config.AutoEnableV2 = func(_, _ string, _ enumspb.TaskQueueType) bool { return true }
```

Comment:
should we use defaultTestConfig instead of NewConfig above? and then we don't need this here if you put AutoEnableV2 in defaultTestConfig.